pkg/pushgateway/config.go

```diff
@@ -41,6 +70,13 @@ func NewConfig(p output.Params) (Config, error) {
 		case "K6_JOB_NAME":
 			cfg.JobName = v
 		}
+		if strings.HasPrefix(k, "K6_LABEL_") {
+			key := strings.ToLower(k[9:])
+			cfg.Labels[key] = strings.ToLower(v)
+		}
+	}
+	if cfg.Debug {
+		fmt.Printf("Pushgateway labels %+v\n", cfg.Labels)
```

Comment:
What do you think if this debug print is done at `Output#Description` as below?

```diff
func (o *Output) Description() string {
-	return "pushgateway: " + o.config.PushGWUrl
+	return fmt.Sprintf("pushgateway: %s, labels: %v", o.config.PushGWUrl. o.config.Labels
}
```

Reply:
Done. It is much better with detailed Description output 🙏